### PR TITLE
Improvements functionality for "Folder as workspace" panel

### PR DESCRIPTION
**Pull request by induction-lab**

### CODE REVIEW

**Comment by sasumner** [comment]
Which issue does this PR pertain to?

**Comment by induction-lab** [comment]
This is not a solution, it is a small extension of the existing functionality.

**Comment by LordNyriox** [comment]
@induction-lab: 

>it is a small extension of the existing functionality. 

Perhaps you can explain *what* you are extending, and *how* you are extending it? 

Your PR title does not clarify anything for me.  :\

File: PowerEditor/src/Parameters.h [hunk 1]
```diff
@@ -1512,6 +1512,8 @@ class NppParameters final
 	};
 
 	const std::vector<generic_string> getFileBrowserRoots() const { return _fileBrowserRoot; };
+    const generic_string getFileBrowserActiveRoot() const { return _fileBrowserActiveRoot; };
```
Comment (AjayVV) [inline PowerEditor/src/Parameters.h]: It is not aligned well.

File: PowerEditor/src/WinControls/FileBrowser/fileBrowser.h [hunk 2]
```diff
@@ -96,7 +98,10 @@ friend class FolderUpdater;
 };
 
 enum BrowserNodeType {
-	browserNodeType_root = 0, browserNodeType_folder = 2, browserNodeType_file = 3
+	browserNodeType_root = 0,
```
Comment (AjayVV) [inline PowerEditor/src/WinControls/FileBrowser/fileBrowser.h]: Not well aligned.

**Comment by MetaChuh** [comment]
@induction-lab 

please read our [rules for pull requests](https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/CONTRIBUTING.md) e.g. opening a feature request issue beforehand, **providing detailed information about any enhancements**, asking for acceptance, before submitting a pr.

many thanks and best regards.

**Pull request closed without merging**